pysteps/extrapolation/semilagrangian.py

```diff
@@ -140,10 +152,10 @@ def interpolate_motion(D, V_inc, td):
         XYW = [XYW[1, :, :], XYW[0, :, :]]
 
         VWX = ip.map_coordinates(
```

Comment:
we could probably benefit from a better variable naming, `XYW`, `VWX` and so on are not very clear. Can we do something about it?

Reply:
Fixed.